docs/manual/annotating-libraries.tex

```diff
@@ -572,6 +572,7 @@ \subsubsection{If you do not have access to the Java source code\label{stub-crea
   must be on your classpath.)
 
 \begin{Verbatim}
+  export CLASSPATH=/home/user_name/checker-framework-2.3.2/checker/dist/checker.jar:javac.jar
```

Comment:
You should use `$JSR308` to refer to the installation path:

`export CLASSPATH=$JSR308/checker-framework/checker/dist/checker.jar`

The `javac.jar` shouldn't be necessary, as javac is contained in `checker.jar`. It would also need a full path to work.